ext/pcntl/tests/pcntl_getpriority_error.phpt

```diff
@@ -8,16 +8,26 @@ pcntl
 if (!function_exists('pcntl_getpriority')) {
     die('skip pcntl_getpriority doesn\'t exist');
 }
+
+if (PHP_OS == "Darwin") {
+    die("skip do not run on Darwin");
```

Comment:
If the test is skipped, "reason: do not run on Darwin" will be shown. The wording looks a bit strange; maybe change to "does not run on Darwin", or "this test is not for Darwin", or something like that.